core/src/initializers/coreVersionValidator.ts

```diff
@@ -0,0 +1,20 @@
+import { Initializer } from "actionhero";
+
+export class CoreVersionValidator extends Initializer {
+  constructor() {
+    super();
+    this.name = "coreVersionValidator";
+    this.startPriority = 999999999999999;
```

Comment:
Turns out Actionhero doesn't like `Infinity`.

Reply:
I *think* that 10000 should be more than enough :D